sys/src/cmd/winwatch.c

```diff
@@ -277,8 +276,11 @@ main(int argc, char **argv)
 	lightblue = allocimagemix(display, DPalebluegreen, DWhite);
 	if(lightblue == nil)
 		sysfatal("allocimagemix: %r");
-	if((font = openfont(display, fontname)) == nil)
-		sysfatal("font '%s' not found", fontname);
+	if(fontname == nil)
+		font = opendefaultfont(display);
```

Comment:
I still don't get why you don't make opendefaultfont take an optional name. You could reduce these lines to one call 
font = opendefaultfont(display, fontname);
done